<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshStatus</class>
 <widget class="QWidget" name="MeshStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Mesh Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="connectionStatusTableWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>356</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Source</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quality</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Destination</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>Ground</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>1.00</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>MAV 001</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="updatePushButton">
        <property name="text">
         <string>Manual Update</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="autoUpdateCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Auto Update</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
